### ISSUE TITLE
Meteor 1.3

### ISSUE THREAD

**Issue by logvik** [VeliovGroup/Meteor-logger-console#1]
When I done update on Meteor 1.3 version errors came:

> Exception while invoking method 'sendProfile' { 
> stack: 'TypeError: Cannot call method \'substring\' of undefined\n  
> at [object Object].Debug.trace (C:\Users\~\AppData\Local\.meteor\packages\ostrio_loggerconsole\1.0.0\npm\node_modules\console-debug\debug.js:97:41)\n  
> at [object Object].Debug.debug (C:\Users\~\AppData\Local\.meteor\packages\ostrio_loggerconsole\1.0.0\npm\node_modules\console-debug\debug.js:157:8)\n  
> at Object.Package (packages\ostrio_loggerconsole.js:77:21)\n  
> at Logger.logit (packages\ostrio_logger.js:92:14)\n  
> at Logger.debug (packages\ostrio_logger.js:214:17)\n  
> at [object Object].sendHandymanProfile (server/methods.js:10:16)\n  
> at [object Object].methodMap.(anonymous function) (packages\meteorhacks_kadira.js:2586:30)\n  
> at maybeAuditArgumentChecks (packages/ddp-server/livedata_server.js:1704:12)\n  
> at packages/ddp-server/livedata_server.js:711:19\n  
> at [object Object]._.extend.withValue (packages\meteor.js:1100:17)',
> I20160330-13:33:00.946(3)?   source: 'method' }

Reinstallation doesn't help, something wrong with packages\ostrio_loggerconsole\1.0.0\npm\node_modules\console-debug\debug.js:97:41

```
var completeMessage = "",
            format = dateFormat(new Date(), 'HH:MM:ss'),
            func = undefined,
            appDir = path.dirname(require.main.filename),
            p = this.get_file_parent();

        if(type != "ERROR"){
            if (p) {
                var parentfile = p.split(appDir)[1].substring(1); // HERE BUG
                func = " @ "+parentfile+":"+this.get_line_parent();
            } else {
                return;
            }
        }
```

**Comment by dr-dimitru** [comment]
It's [console-debug](https://www.npmjs.com/package/console-debug) issue. Make sure you're passing all arguments.
Please, wait until I upgrade and test this package on Meteor 1.3

**Comment by dr-dimitru** [comment]
Hi @logvik , thank you for contribution

This issue is fixed in [v1.1.0](https://github.com/VeliovGroup/Meteor-logger-console/releases/tag/1.1.0), please confirm on your end

**Comment by logvik** [comment]
Thank you! I confirm that everything is ok now.
